docs/snapshots.md

```diff
@@ -0,0 +1,51 @@
+# Index snapshots
```

Comment:
```suggestion
# Index Snapshots
```

See https://en.wikipedia.org/wiki/Title_case#Chicago_Manual_of_Style

Reply:
Sorry, I must be so annoying 😅 This is at least the third time you're correcting me for the same thing. Fixing!